backend/app/converters/ead_converter.rb

```diff
@@ -204,50 +204,113 @@ def self.configure
     end
 
 
+    def make_single_note(note_name, tag, tag_name="")
+      content = tag.inner_text
+      if !tag_name.empty?
+        content = tag_name + ": " + content
+      end
+      make :note_singlepart, {
+        :type => note_name,
+        :persistent_id => att('id'),
+        :content => format_content( content.sub(/<head>.?<\/head>/, '').strip)
+      } do |note|
+        set ancestor(:resource, :archival_object), :notes, note
+      end
+    end
+    
+    def make_nested_note(note_name, tag)
+      content = tag.inner_text
+    
+      make :note_multipart, {
+        :type => note_name,
+        :persistent_id => att('id'),
+        :subnotes => {
+          'jsonmodel_type' => 'note_text',
+          'content' => format_content( content )
+        }
+      } do |note|
+        set ancestor(:resource, :archival_object), :notes, note
+      end
+    end
+    
     with 'physdesc' do
       physdesc = Nokogiri::XML::DocumentFragment.parse(inner_xml)
+    
       extent_number_and_type = nil
+    
+      dimensions = []
+      physfacets = []
+      container_summaries = []
       other_extent_data = []
-      make_note_too = false
+    
+      container_summary_texts = []
+      dimensions_texts = []
+      physfacet_texts = []
+    
+      # If there is already a portion of 'part' specified, use it
+      if att('altrender').downcase == 'part'
```

Comment:
@walkerdb - Looks like a test failed due to this line - 
https://travis-ci.org/archivesspace/archivesspace/jobs/92198055
Maybe just change to `if att('altrender') && att('altrender').downcase == 'part'`?